aspnetcore/fundamentals/configuration/index.md

```diff
@@ -116,7 +158,12 @@ For more information on storing passwords or other sensitive data:
 
 <a name="evcp"></a>
 
-## Environment variables
+## Non-prefixed environment variables
+
+Non-prefixed environment variables are environment variables other than those prefixed by `ASPNETCORE_` or `DOTNET_`. For example, the ASP.NET Core web application templates set `"ASPNETCORE_ENVIRONMENT": "Development"` in `launchSettings.json`. For more information on `ASPNETCORE_` and `DOTNET_` environment variables, see:
+
+* <xref:fundamentals/environments>
+* [`DOTNET_` environment variables](/dotnet/core/tools/dotnet-environment-variables)
```

Comment:
`<xref:fundamentals/environments>` looks to be specifically about prod/dev/staging environments. While this can be configured using `DOTNET_` or `ASPNETCORE_` environment variables, it's not about environment variables themselves.

```suggestion
* [List of highest to lowest priority default configuration sources](#hi2low) including non-prefixed, `ASPNETCORE_`-prefixed and `DOTNETCORE_`-prefixed environment variables.
* [`DOTNET_` environment variables](/dotnet/core/tools/dotnet-environment-variables) used outside of [Microsoft.Extensions.Hosting](xref:Microsoft.Extensions.Hosting).
```